core/chains/solana/soltxm/txm_test.go

```diff
@@ -0,0 +1,54 @@
+package soltxm_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	solanaGo "github.com/gagliardetto/solana-go"
+	solanaClient "github.com/smartcontractkit/chainlink-solana/pkg/solana/client"
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/config"
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/db"
+	"github.com/smartcontractkit/chainlink/core/chains/solana/soltxm"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTxm_HappyPath(t *testing.T) {
+	received := make(chan struct{})
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
hm likewise with the client_test.go is there no way to test against a real solana node here and assert balances change etc.?